# Exception Handling in C++
try: represents a block of code that can throw an exception
catch: represents a block of code that is executed when a particular exception is thrown.
throw: Used to throw an exception. Also used to list the exceptions that a function throws, but doesn't handle itself.

## Why Exception Handling?
Following are main advantages of exception handling over traditional error handling.
- Seperation of Error Handling code from Normal Code
- Functions/Methods can handle any exceptions they choose
- Grouping of Error Types
